eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -268,7 +269,7 @@ object NodeParams extends Logging {
     val nodeAlias = config.getString("node-alias")
     require(nodeAlias.getBytes("UTF-8").length <= 32, "invalid alias, too long (max allowed 32 bytes)")
 
-    def validateFeatures(features: Features): Unit = {
+    def validateFeatures(features: Features[InitFeature]): Unit = {
```

Comment:
Do you really need to make this `InitFeature`? There's no reason that it will be restricted to `init`, this function makes sense for all feature types, it should be `FeatureScope`

Reply:
Done